app/model/SerialUpdateResult.scala

```diff
@@ -37,24 +36,18 @@ case class SerialUpdateResult[UpdateRequest](
 object SerialUpdateResult {
 
   object Result extends Enumeration {
-    val SUCCESS, FAILURE, PARTIAL = Value
+    val SUCCESS, FAILURE, PARTIAL, NO_OP = Value
   }
 
   def fromEither[T](results: Seq[Either[T, T]]): SerialUpdateResult[T] = {
 
-    results.foldLeft(List.empty[T], List.empty[T]){ case (acc, res) =>
+    val (f, s) = results.foldLeft(List.empty[T], List.empty[T]){ case (acc, res) =>
       res match {
         case Left(l) => (acc._1 :+ l, acc._2)
         case Right(r) => (acc._1, acc._2 :+ r)
```

Comment:
Would deconstructing the ```acc``` into the two constituent parts make this rather obscure class more readable?